packages/native-stack/src/types.tsx

```diff
@@ -330,6 +335,12 @@ export type NativeStackNavigationOptions = {
    * @platform ios
    */
   autoHideHomeIndicator?: boolean;
+  /**
+   * Whether the keyboard should hide when swiping to the previous screen. Defaults to `false`.
+   *
+   * @platform ios
+   */
+  hideKeyboardOnSwipe?: boolean;
```

Comment:
JS stack has https://reactnavigation.org/docs/stack-navigator/#keyboardhandlingenabled so let's align that if possible

Reply:
Good idea 👍